hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -1456,6 +1456,20 @@
       en_run_modes: ["sw_test_mode_test_rom"]
       run_opts: ["+en_scb_tl_err_chk=0", "+use_jtag_dmi=1"]
     }
+    {
+      name: chip_sw_rv_dm_ndm_reset_req_when_cpu_halted
+      uvm_test_seq: "chip_sw_rv_dm_ndm_reset_when_cpu_halted_vseq"
+      sw_images: ["//sw/device/tests/sim_dv:rv_dm_ndm_reset_req_when_cpu_halted:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+use_jtag_dmi=1"]
+    }
+    {
+      name: chip_sw_rv_dm_access_after_escalation_reset
+      uvm_test_seq: "chip_sw_rv_dm_access_after_escalation_reset_vseq"
+      sw_images: ["//sw/device/tests/sim_dv:alert_handler_escalation_test:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+use_jtag_dmi=1"]
+    }
```

Comment:
Should we run these both in `DEV` and `RMA` or `TEST_UNLOCKED*` ? 
Just to make sure the differing strap sampling behavior does not break anything.

Reply:
That part seems to be adequately tested in other tests. Do you think this is necessary?